main/src/library/flix/actor/SystemEvent.flix

```diff
@@ -36,13 +36,20 @@ pub enum SystemEvent {
     /// A command that the actor system should start.
     /// TODO: should there be a Channel[Unit] to notify that the start has completed as well?
     ///
-    case SystemStart
+    case SystemStart,
 
     ///
     /// A command that the actor system should shutdown.
     ///
     /// @param the channel on which the `Unit` value is sent once the shutdown has completed.
     ///
-    case SystemShutdown(Channel[Unit])
+    case SystemShutdown(Channel[Unit]),
 
+    ///
+    /// A command to retrieve the reference to a running actor.
+    ///
+    /// @param the name of the actor
+    /// @param the channel on which the reference to the actor is sent. If the actor is not found, then None will be sent.
+    /// 
+    case GetActor(Str, Channel[Option[ActorRef]])
```

Comment:
I assume that this is to communicate with actors from outside the system, but is that really desirable?

Reply:
Afterall, what is the state of an actor outside of the system?